daemon/api.go

```diff
@@ -1748,17 +1790,18 @@ func postCreateUser(c *Command, r *http.Request, user *auth.UserState) Response
 		return BadRequest("cannot use create-user as non-root")
 	}
 
-	var createData struct {
-		Email  string `json:"email"`
-		Sudoer bool   `json:"sudoer"`
-		Known  bool   `json:"known"`
-	}
+	var createData postUserCreateData
 
 	decoder := json.NewDecoder(r.Body)
 	if err := decoder.Decode(&createData); err != nil {
 		return BadRequest("cannot decode create-user data from request body: %v", err)
 	}
 
+	// special case: the user requested the creation of all known
+	// system-users
+	if createData.Email == "" && createData.Known {
+		return createAllKnownSystemUsers(c.d.overlord.State(), &createData)
```

Comment:
It would be better to have something reflecting more closely the CLI API:

`CreateUser(&CreateUserOptions{Email: ..., Known: true/false})`